docs/build/configure/_index.md

```diff
@@ -231,16 +231,31 @@ Find more information in the [processes documentation](/build/configure/processe
 
 ## Webhooks
 
-Webhooks allow you to trigger actions when certain types of data are sent from your machine to the cloud.
-For example, you can configure a webhook to send you a notification when your robot's sensor collects a new reading.
+Webhooks allow you to trigger actions when certain types of data are sent from your machine to the cloud, or when the internet connectivity of your machine changes.
```

Comment:
Hmm so I think it still makes sense to provide the concrete example up top, before we get into the different types.
```suggestion
Webhooks allow you to trigger actions when certain types of data are sent from your machine to the cloud, or when the internet connectivity of your machine changes.
For example, you can configure a webhook to send you a notification when your robot's sensor collects a new reading.
```